modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/model/listener/PortalInstanceLifecycleListenerImpl.java

```diff
@@ -46,6 +54,7 @@ public void portalInstanceRegistered(Company company) throws Exception {
 			_commerceCurrencyLocalService, company);
 	}
 
-	private final CommerceCurrencyLocalService _commerceCurrencyLocalService;
+	@Reference
```

Comment:
@JanisZhang You should do this commit together with the removal of registration logic inside CommerceCurrencyLocalServiceImpl, this way, people will get what you are doing directly. It confuses me why you are removing the registration logic until I found the conversation between you and Dante.